docs/guides/drivers/docker-container.md

```diff
@@ -1,40 +1,42 @@
 # Docker container driver
 
-The buildx docker-container driver allows creation of a managed and
-customizable BuildKit environment inside a dedicated Docker container.
+The buildx Docker container driver allows creation of a managed and customizable
+BuildKit environment in a dedicated Docker container.
 
-Using the docker-container driver has a couple of advantages over the basic
-docker driver. Firstly, we can manually override the version of buildkit to
-use, meaning that we can access the latest and greatest features as soon as
-they're released, instead of waiting to upgrade to a newer version of Docker.
-Additionally, we can access more complex features like multi-architecture
-builds and the more advanced cache exporters, which are currently unsupported
-in the default docker driver.
+Using the Docker container driver has a couple of advantages over the default
+Docker driver. For example:
 
-We can easily create a new builder that uses the docker-container driver:
+- Specify custom BuildKit versions to use.
+- Build multi-arch images
+- Advanced options for [cache import and export](../cache/index.md)
```

Comment:
```suggestion
- Advanced options for [cache import and export](index.md)
```

Reply:
I wanted to link to the cache docs overview. Bad idea? :)

Reply:
> I wanted to link to the cache docs overview.

Ah I thought your were linking to the same folder.

> Bad idea? :)

Yes atm we can't link to pages outside the current folder for remote pages. You have to use absolute links:

```suggestion
- Advanced options for [cache import and export](https://docs.docker.com/build/building/cache/)
```